pyoverkiz/enums/general.py

```diff
@@ -126,6 +126,7 @@ class FailureType(IntEnum):
     PRIORITY_LOCK__NO_SECURITY_DEVICE = 192
     PRIORITY_LOCK__DEAD_SENSOR = 193
     PRIORITY_LOCK__UNKNOWN_ERROR = 194
+    CMDDEPRECATED = 197
```

Comment:
Oh! Nice to see they start to depreciate some commands. In which case did you see it?

Reply:
https://github.com/home-assistant/core/issues/91206, but not more context than this :(

Reply:
Hope we will get an answer.